server/src/controllers/__tests__/users.js

```diff
@@ -125,6 +125,6 @@ test('deleteUser will 403 if not made by the author', async () => {
   expect(res.status).toHaveBeenCalledTimes(1)
   expect(res.status).toHaveBeenCalledWith(403)
   expect(res.send).toHaveBeenCalledTimes(1)
-  const userFromDb = await db.getPost(testUser.id)
+  const userFromDb = await db.getUser(testUser.id)
```

Comment:
Uh oh... Looks like I need to fix this (it's failing in CI). I'll give this a look later. Thanks!

Reply:
hm, it works on my local.... is it something we need to update in db?